### ISSUE TITLE
hardcoded download directory auto_pahe.py

### ISSUE THREAD

**Issue by Mistymimi** [haxsysgit/autopahe#2]
In auto_pahe.py a hardcoded download directory is in two places. I believe its using a linus or max os directory for your machine. If I changed both to a windows directory it gives a unicode error. Both errors stop the program from working as intended.

**Comment by haxsysgit** [comment]
yes i'm using a linux machine , so as per your request i'll have to make it generally acceptable
